### ISSUE TITLE
Missing Gradle build file?

### ISSUE THREAD

**Issue by bachi55** [kaibioinfo/computefp#1]
Hei,

could you add a 'build.gradle' file to the repository? 

Best regards,
Eric

**Comment by kaibioinfo** [comment]
Solved by #725a1db61da5ace506306aca414dbdbe456e9460

**Comment by bachi55** [comment]
Hei, 

unfortunately I receive the follow error when using `gradle distInstall`. It seems that a couple of  dependencies cannot be found, such as: `jmzml` (in version 1.7.11), `cpdetector`, ... I added them manually to the `build.gradle` file:

```
repositories {
    mavenLocal()
    mavenCentral()
    maven {//public libs only so anybody can build the frontend without login
        url "https://bio.informatik.uni-jena.de/repository/libs-oss"
    }
    maven {
        url "https://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo"
    }
    maven {
        url "https://www.ebi.ac.uk/intact/maven/nexus/content/repositories/ebi-repo/"
    }
    maven {
        url "https://mvnrepository.com/artifact/antlr/antlr"
    }
}

dependencies {
    compile group: 'uk.ac.ebi.jmzml', name: 'jmzml', version: '1.7.11'
    compile group: 'cpdetector', name: 'cpdetector', version: '1.0.7'
    compile "de.unijena.bioinf.ms:fingerprinter_oss:1.3.0-SNAPSHOT"
    compile "de.unijena.bioinf.ms:chemistry_base:4.3.0-SNAPSHOT"
}
```
However, still I receive "missing dependency" exceptions for other packages: 

```
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find org.antlr:com.springsource.antlr:2.7.7.
     Searched in the following locations:
       - file:/home/bach/.m2/repository/org/antlr/com.springsource.antlr/2.7.7/com.springsource.antlr-2.7.7.pom
       - https://repo.maven.apache.org/maven2/org/antlr/com.springsource.antlr/2.7.7/com.springsource.antlr-2.7.7.pom
       - https://bio.informatik.uni-jena.de/repository/libs-oss/org/antlr/com.springsource.antlr/2.7.7/com.springsource.antlr-2.7.7.pom
       - https://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/org/antlr/com.springsource.antlr/2.7.7/com.springsource.antlr-2.7.7.pom
       - https://www.ebi.ac.uk/intact/maven/nexus/content/repositories/ebi-repo/org/antlr/com.springsource.antlr/2.7.7/com.springsource.antlr-2.7.7.pom
       - https://mvnrepository.com/artifact/antlr/antlr/org/antlr/com.springsource.antlr/2.7.7/com.springsource.antlr-2.7.7.pom
     Required by:
         project :
         project : > cpdetector:cpdetector:1.0.7
   > Could not find net.sourceforge.cpdetector:cpdetector:1.0.7.
     Searched in the following locations:
       - file:/home/bach/.m2/repository/net/sourceforge/cpdetector/cpdetector/1.0.7/cpdetector-1.0.7.pom
       - https://repo.maven.apache.org/maven2/net/sourceforge/cpdetector/cpdetector/1.0.7/cpdetector-1.0.7.pom
       - https://bio.informatik.uni-jena.de/repository/libs-oss/net/sourceforge/cpdetector/cpdetector/1.0.7/cpdetector-1.0.7.pom
       - https://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/net/sourceforge/cpdetector/cpdetector/1.0.7/cpdetector-1.0.7.pom
       - https://www.ebi.ac.uk/intact/maven/nexus/content/repositories/ebi-repo/net/sourceforge/cpdetector/cpdetector/1.0.7/cpdetector-1.0.7.pom
       - https://mvnrepository.com/artifact/antlr/antlr/net/sourceforge/cpdetector/cpdetector/1.0.7/cpdetector-1.0.7.pom
     Required by:
         project : > uk.ac.ebi.jmzml:jmzml:1.7.11 > uk.ac.ebi.pride.architectural:pride-xml-handling:1.0.3 > psidev.psi.tools:xxindex:0.23

```

Could you indicate how to set the dependencies properly? 

Best regards,

Eric
